raml-parser-2/src/main/java/org/raml/v2/internal/impl/commons/model/type/TypeDeclaration.java

```diff
@@ -90,6 +89,42 @@ public String type()
         return getTypeExpression(getTypeNode());
     }
 
+    public List<TypeDeclaration> parentTypes() {
+
+        List<TypeDeclaration> nodes =  new ArrayList<>();
+        Set<String> parentNames = new HashSet<>();
+        Node parents =  resolvedType.getTypeDeclarationNode().get("type");
+        if ( parents == null) {
+
+           return nodes;
```

Comment:
if there's no explicit supertype then parentTypes should return the corresponding builtin type